DataFormats/TrackReco/src/classes_def.xml

```diff
@@ -1,11 +1,12 @@
 <lcgdict>
-  <class name="std::bitset<reco::TrackBase::algoSize>"/>
+  <class name="reco::TrackBase::AlgoMask"/>
   <class name="reco::HitPattern" ClassVersion="12">
       <version ClassVersion="12" checksum="3922863495"/>
       <version ClassVersion="11" checksum="1621684703"/>
   </class>
 
-  <class name="reco::TrackBase" ClassVersion="13">
+  <class name="reco::TrackBase" ClassVersion="15">
+   <version ClassVersion="15" checksum="3673246687"/>
    <version ClassVersion="14" checksum="3929365050"/>
    <version ClassVersion="13" checksum="1244921154"/>
```

Comment:
does it matter that the checksum is different here compared to 75X?
https://github.com/cms-sw/cmssw/blob/CMSSW_7_5_X/DataFormats/TrackReco/src/classes_def.xml#L10
```xml
  <class name="reco::TrackBase" ClassVersion="15">
   <version ClassVersion="15" checksum="1802760569"/>
   <version ClassVersion="14" checksum="3929365050"/>
```